gwt-ol3-client/src/main/java/ol/MapOptions.java

```diff
@@ -32,49 +35,110 @@
 @JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
 public class MapOptions implements Options {
 
+    /**
+     * @param controls Controls initially added to the map. If not specified, <code>ol.control.defaults()</code> is used.
+     */
     @JsProperty
-    public native double getPixelRatio();
+    public native void setControls(Collection<Control> controls);
 
     @JsProperty
-    public native String getTarget();
+    public native Collection<Control> getControls();
 
+	/**
+	 * @param pixelRatio The ratio between physical pixels and device-independent pixels (dips) on the device.
+	 * If <code>undefined</code> then it gets set by using <code>window.devicePixelRatio</code>.
+	 */
     @JsProperty
-    public native void setTarget(String target);
+    public native void setPixelRatio(Double pixelRatio);
```

Comment:
Agree. Moreover, because of [JsInterop limitations](https://github.com/gwtproject/gwt/issues/9286#issuecomment-186375048) we can't use Integer.

Will push changes soon.